jetstream/statistics.py

```diff
@@ -359,10 +364,12 @@ def transform(self, df: DataFrame, metric: str) -> StatisticResultCollection:
 
 
 class Count(Statistic):
-    def apply(self, df: DataFrame, metric: str):
+    def apply(self, df: DataFrame, metric: str, reference_branch: Optional[str]):
         return self.transform(df, metric)
 
-    def transform(self, df: DataFrame, metric: str) -> StatisticResultCollection:
+    def transform(
+        self, df: DataFrame, metric: str, reference_branch: str = "control"
```

Comment:
Should `reference_branch` have a default? The others don't seem to.

Reply:
Right, probably better to stay consistent here